airbyte-integrations/connectors/source-chargebee/source_chargebee/schemas/comment.json

```diff
@@ -28,7 +28,19 @@
       "type": ["string", "null"]
     },
     "custom_fields": {
-      "type": ["array", "null"]
+      "type": ["null", "array"],
+      "items": {
+        "type": ["null", "object"],
+        "additionalProperties": true,
+        "properties": {
+          "name": {
+            "type": ["null", "string"]
+          },
+          "value": {
+            "type": ["null", "string"]
+          }
+        }
+      }
```

Comment:
please move custom field to `schema/shared` folder and make a ref to it in all schemas.

Reply:
Moved to shared/_definitions.json and references added in each schema